### PR TITLE
First pass, GH actions

### PR DESCRIPTION
**Pull request by wasade**

### CODE REVIEW

File: .github/workflows/python-package-conda.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+name: microsetta-private-api CI
+    
+on: 
+  push:
+    branches: [ master ]
+  pull_request:
+
```
Comment (dhakim87) [inline .github/workflows/python-package-conda.yml:7]: Does this need a branches: [ master ] as well?  Or any PR to any branch will build and run tests?

File: .github/workflows/python-package-conda.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # based on https://github.com/actions/example-services/blob/master/.github/workflows/postgres-service.yml#L44-L72
+          - 5432/tcp
+
+    steps:
+      # Downloads a copy of the code in your repository before running CI tests
+      - name: Check out repository code
+        uses: actions/checkout@v2
```
Comment (dhakim87) [inline .github/workflows/python-package-conda.yml:37]: What is the @v2 here?

File: microsetta_private_api/config_manager.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 class DBConfig(object):
     def __init__(self):
         self.user = 'postgres'
-        self.password = ''
+        self.password = 'postgres'
```
Comment (dhakim87) [inline microsetta_private_api/config_manager.py:18]: Intentional change to default localhost postgres server password?

**Review by dhakim87 — COMMENTED** [review]
I admit I don't really know a ton about GH actions.  Everything seems legit- I've sprinkled a few questions throughout.

File: .github/workflows/python-package-conda.yml (hunk 1 above)
Comment (wasade) [reply .github/workflows/python-package-conda.yml:7]: Not fully sure, just using templates right now

File: .github/workflows/python-package-conda.yml (hunk 2 above)
Comment (wasade) [reply .github/workflows/python-package-conda.yml:37]: I believe this refers back to a version of the github action definition from the github actions repository itself. But, not very sure, just pulling from templates

File: microsetta_private_api/config_manager.py (hunk 3 above)
Comment (wasade) [reply microsetta_private_api/config_manager.py:18]: I don't know what the default is for the build of postgres used

**Comment by wasade** [comment]
Thanks!!! And ya, I don't know a ton either... but it seems to run :)

**Pull request merged**